sql/ll_bitemporal_inactivate_select.sql

```diff
@@ -0,0 +1,97 @@
+ CREATE OR REPLACE FUNCTION bitemporal_internal.ll_bitemporal_inactivate_select(p_schema_name text,
+  p_table_name text
+,p_search_fields TEXT  -- search fields
+,p_values_selected_search TEXT  --  search values selected
+, p_effective temporal_relationships.timeperiod -- inactive starting
+, p_asserted temporal_relationships.timeperiod -- will be asserted
+)
+RETURNS INTEGER
+AS
+$BODY$
+DECLARE
+v_sql text ;
+v_rowcount INTEGER:=0;
+v_list_of_fields_to_insert text:=' ';
+v_list_of_fields_to_insert_excl_effective text;
+v_table_attr text[];
+v_now timestamptz:=now();-- so that we can reference this time
+  v_keys int[];
+  v_keys_old  int[];
+  v_serial_key text:=p_table_name||'_key';
+  v_table text:=p_schema_name||'.'||p_table_name;
+ 
+BEGIN 
+ IF lower(p_asserted)<v_now::date --should we allow this precision?...
+    OR upper(p_asserted)< 'infinity'
+ THEN RAISE EXCEPTION'Asserted interval starts in the past or has a finite end: %', p_asserted
```

Comment:
Should we make some helper functions to assert conditions on the intervals?
I think would make it easier to read. 

Also are the fst and snd functions too confusing? I personally don't like lower & upper as names for the ranges and find it confusing but I think that might just be me. I have been wondering if "start_point" and "end_point" might be more readable?